eth/gasprice/gasprice.go

```diff
@@ -299,53 +270,48 @@ func (oracle *Oracle) SuggestPrice(ctx context.Context) (*big.Int, error) {
 // necessary to add the basefee to the returned number to fall back to the legacy
 // behavior.
 func (oracle *Oracle) SuggestTipCap(ctx context.Context) (*big.Int, error) {
-	tip, _, err := oracle.suggestDynamicFees(ctx)
-	return tip, err
+	return oracle.suggestTip(ctx)
 }
 
-// suggestDynamicFees estimates the gas tip and base fee based on a simple sampling method
-func (oracle *Oracle) suggestDynamicFees(ctx context.Context) (*big.Int, *big.Int, error) {
+// suggestTip estimates the gas tip based on a simple sampling method
+func (oracle *Oracle) suggestTip(ctx context.Context) (*big.Int, error) {
```

Comment:
coreth has it as `suggestDynamicFees` (as it was before here as well it seems), why the rename?

Reply:
this is renamed in https://github.com/ava-labs/coreth/pull/865